test/hotspot/jtreg/vmTestbase/nsk/jvmti/ResourceExhausted/resexhausted003/TestDescription.java

```diff
@@ -39,10 +39,7 @@
  *          /test/lib
  * @run main/othervm/native
  *      -agentlib:resexhausted=-waittime=5
- *      -Xms64m
- *      -Xmx64m
```

Comment:
Don't you need these to ensure we get OOME in a reasonable amount of time on systems with huge memories?

Reply:
The test trigger OOME in Metaspace only by loading classes, so heap shouldn't be used.

Reply:
Ah I see.